tests/update_reference_data.sh

```diff
@@ -54,14 +56,37 @@ for test_name in ${@:2}; do
       SPE9_CP_SHORT \
       EGRID INIT PRT SMSPEC UNRST UNSMRY
   fi
-done
-
-
 
+  if grep -q "norne_init" <<< $test_name
+  then
+    copyToReferenceDir \
+      $configuration/build-opm-simulators/tests/results/init/flow+norne/ \
+      $OPM_DATA_ROOT/norne/opm-simulation-reference/ \
+      NORNE_ATW2013 \
+      EGRID INIT
+  fi
+done
 
+if [ -z "${@:2}" ]
+then
+  # User did not specify tests to update, probe
```

Comment:
the update_reference_data.sh script allows specifying which tests to update. to avoid this hassle in the jenkins trigger, this probes for which tests were changed by looking at which files are changed in the diff.